src/macaron/slsa_analyzer/checks/check_result.py

```diff
@@ -29,6 +25,32 @@ class CheckResultType(str, Enum):
     UNKNOWN = "UNKNOWN"
 
 
+class Confidence(float, Enum):
+    """This class contains confidence score for a check result.
+
+    The scores must be in [0.0, 1.0].
+    """
+
+    #: A high confidence score.
+    HIGH = 1.0
+
+    #: A medium confidence score.
+    MEDIUM = 0.7
+
+    #: A low confidence score.
+    LOW = 0.5
```

Comment:
I wonder how you picked these values.

Reply:
The values themselves are not important. The important part is having three levels, which can definitely change if needed.

Reply:
Hmm, then my follow-up question would be why do we need 3 levels? I assume the answer is likely as simple as "We define a few set levels just so that we do not have to think about different arbitrary confidence score values when writing checks."